media-video/mplayer/mplayer-1.4.0.recipe

```diff
@@ -193,6 +191,7 @@ BUILD()
 		--enable-haiku \
 		--enable-menu \
 		--enable-runtime-cpudetection \
+		--enable-openssl-nondistributable \
```

Comment:
"nondistributable"?

Reply:
https://people.gnome.org/~markmc/openssl-and-the-gpl.html

Can mplayer use another SSL library instead?